indigo_app/views/workflows.py

```diff
@@ -56,12 +56,12 @@ def get_context_data(self, *args, **kwargs):
 
         tasks = self.object.tasks.all()
         context['has_tasks'] = bool(tasks)
-        context['task_groups'] = Task.task_columns(['open', 'pending_review'], tasks)
+        context['task_groups'] = Task.task_columns(['open', 'pending_review', 'assigned'], tasks)
         context['possible_tasks'] = self.place.tasks.unclosed().exclude(pk__in=[t.id for t in self.object.tasks.all()]).all()
 
         # stats
-        self.object.n_tasks = self.object.tasks.count()
-        self.object.n_done = self.object.tasks.closed().count()
+        self.object.n_tasks = self.object.tasks.count() - self.object.tasks.filter(state='cancelled').count()
```

Comment:
I think this is being too smart. If a cancelled task is part of a workflow, it should be counted as such. It's still a useful thing to know if a task has been cancelled -- it says "we're not doing that" and there's an historical record. I see that as different from a task being deleted.

Reply:
gotcha. I'll change this back.